<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_login_window</class>
 <widget class="QMainWindow" name="customer_login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer Login Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="csloginwdw_lbl_heading">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Customer Login </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="csloginwdw_lbl_ADid">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Customer ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="csloginwdw_lbl_ADpassword">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csloginwdw_btn_login">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="csloginwdw_linedit_ADid">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="csloginwdw_linedit_ADpassword">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="csloginwdw_btn_returnmain">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Main Window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csloginwdw_btn_exit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
